test/e2e/webdriver/driver.js

```diff
@@ -505,7 +505,17 @@ class Driver {
 
   // Error handling
 
-  async verboseReportOnFailure(title) {
+  async verboseReportOnFailure(title, error) {
+    if (process.env.CIRCLECI) {
+      console.error(
+        `Failure in ${title}, for more information see the artifacts tab in CI\n`,
+      );
+    } else {
+      console.error(
+        `Failure in ${title}, for more information see the test-artifacts folder\n`,
+      );
+    }
+    console.error(`error: ${error.message}\n`);
```

Comment:
I can't decide if this line is better as
`console.error('error:', error);`

It then shows the stack trace, which would normally be better, but the line numbers are all wrong.

Reply:
I still think the stack trace might be more beneficial, its still mroe information than just the message

Reply:
Do you have any inkling **why** the line numbers are wrong?  Minify or browserify?